### ISSUE TITLE
Set custom server?

### ISSUE THREAD

**Issue by mankinskin** [rustwasm/wasm-pack#963]
## 💡 Feature description
I would like to system test my own server and Wasm client package together. wasm-pack provides a lot of what I need, but I would like to use my own server instead of the internal one.

#### 💻 Basic example
I can imagine something like this:
```sh
my-server --bind https://0.0.0.0:8000 &
wasm-pack test --server https://0.0.0.0:8000 --headless --firefox
```
So basically we allow wasm-pack to redirect requests to a specified host that should be started beforehand.

**Comment by vdods** [comment]
Yeah, seconded.  I'd like to be able to configure the test server, in particular to set CORS headers (and I'm sure more stuff later), and it seems like this would be the general way to do this (and more).
